packages/gatsby-plugin-manifest/src/gatsby-node.js

```diff
@@ -220,11 +220,9 @@ const makeManifest = async ({
     async function processIconSet(iconSet) {
       // if cacheBusting is being done via url query icons must be generated before cache busting runs
       if (cacheMode === `query`) {
```

Comment:
```suggestion
      if (cacheMode === `query` || cacheMode !== `query`) {
```

Reply:
The other ```if(cacheMode !== `query`)``` statement can be removed by changing the condition as above, because both of them are doing the same thing right?

Reply:
but then this will run always :/